Miscellaneous/docker/full_stack_example/docker-compose-windows.yml

```diff
@@ -0,0 +1,174 @@
+---
+version: "2.1"
+services:
```

Comment:
I'm -1 on os dependent yml files. 

I'm not following the reason why we need separate ones.